tests/mocha/unit-tests/components/Help/help.container.spec.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import { expect } from 'chai'
+import { mount } from 'enzyme'
+import request from 'superagent'
+
+let mock = require('superagent-mocker')(request)
+mock.timeout = 1
+
+import { HelpContainer } from '../../../../../src/assets/js/react/components/Help/HelpContainer'
+
+const state = { searchInputA: 'form', searchInputB: 'forrp' }
+const fixResultsA = {
+  body: [
+    {
+      link: "https://gravitypdf.com/documentation/v5/user-hide-form-fields/",
+      title: { rendered: "Hide Form Fields" },
+      excerpt: { rendered: "<p>Only certain form data is important to you. That&#8217;s why Gravity PDF has a number of ways to filter out the unimportant fields in your generated PDF. It&#8217;s important to&#8230;</p>" }
+    },
+    {
+      link: "https://gravitypdf.com/documentation/v5/user-gravity-forms-compatibility/",
+      title: { rendered: "Gravity Forms Compatibility" },
+      excerpt: { rendered: "<p>Gravity PDF is a third party extension for Gravity Forms. The company who builds Gravity PDF, Blue Liquid Designs, is an independent third party who has no control over Gravity&#8230;</p>" }
+    }
+  ],
+}
+const fixResultsB = {
+  body: []
+}
+
+
+describe('<HelpContainer />', () => {
+
+ beforeEach(function () {
+  mock.clearRoutes()
+ })
+
+ afterEach(function () {
+  mock.clearRoutes()
+ })
+
+ it(`Should fetch data from API and return 'Available result'`, async () => {
+  mock.get(`https://gravitypdf.com/wp-json/wp/v2/v5_docs/?search=${state.searchInputA}`, (req) => {
+    return fixResultsA
+  })
+  const wrapper  = mount(<HelpContainer helpResult={fixResultsA.body} />)
```

Comment:
By passing `helpResult` you're pre-rendering the results. What should happen instead is the results should be empty, you call your function/method that handles the rendering, the mocked API request is triggered, which should then render the results.

Reply:
fixed